code/game/machinery/iv_drip.dm

```diff
@@ -347,7 +341,9 @@
 /obj/machinery/iv_drip/plumbing/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/plumbing/iv_drip, anchored)
-
+	AddComponent(/datum/component/simple_rotation)
+/obj/machinery/iv_drip/plumbing/AltClick(mob/user)
```

Comment:
Don't leave do-nothing overrides please. The comment doesn't really tell me anything.